recipes/SSp_Detector/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "SSp_Detector" %}
+{% set version = "2023.10.25rc1" %}
```

Comment:
This still falls under CFEP-05. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md

You can ask upstream for a stable release or publish under a dev label. See the instructions for the latter in the doc above.

Reply:
Version number updated for stable release